manim/utils/space_ops.py

```diff
@@ -182,6 +270,18 @@ def rotation_matrix(angle: Union[int, float], axis: np.ndarray) -> np.ndarray:
 
 
 def rotation_about_z(angle: Union[int, float]) -> List[Union[int, float]]:
+    """Rotates an angle about z axis
+
+    Parameters
+    ----------
+    angle : Union[int, float]
+        The angle to be rotated.
+
+    Returns
+    -------
+    List[Union[int, float]]
```

Comment:
only float, see here:
https://stackoverflow.com/questions/50928592/mypy-type-hint-unionfloat-int-is-there-a-number-type